Composer/packages/extensions/obiformeditor/src/Form/widgets/ExpressionWidget.tsx

```diff
@@ -23,24 +23,27 @@ interface ExpresionWidgetProps extends ITextFieldProps {
   editable?: boolean;
 }
 
-const getDefaultErrorMessage = () => {
-  return formatMessage.rich('Invalid expression syntax. Refer to the syntax documentation<a>here</a>', {
-    a: ({ children }) => (
-      <a
-        key="a"
-        href="https://github.com/microsoft/BotBuilder-Samples/blob/master/experimental/common-expression-language/prebuilt-functions.md"
-        target="_blank"
-        rel="noopener noreferrer"
-      >
-        {children}
-      </a>
-    ),
-  });
+const getDefaultErrorMessage = (errMessage: string) => {
+  return formatMessage.rich(
+    // eslint-disable-next-line format-message/literal-pattern
```

Comment:
Instead of doing this, use the existing mechanisms of `formatMessage` to pass in a variable.